components/library/goffice/Makefile

```diff
@@ -36,28 +35,40 @@ include $(WS_MAKE_RULES)/ips.mk
 PATH=$(PATH.gnu)
 
 COMPONENT_PREP_ACTION =        ( cd $(@D)  && \
-					libtoolize --copy --force &&\
-					aclocal -I . &&\
-					automake -c -f -a &&\
-					autoconf )
+                    libtoolize --copy --force &&\
+                    aclocal -I . &&\
+                    automake -c -f -a &&\
+                    autoconf )
+
+CFLAGS += $(XPG6MODE) -std=c99
 
-CONFIGURE_OPTIONS+=	--sysconfdir=/etc
-CONFIGURE_OPTIONS+=	--libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
+CONFIGURE_OPTIONS+= --sysconfdir=/etc
+CONFIGURE_OPTIONS+= --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
 
 CONFIGURE_ENV+= PYTHON=$(PYTHON)
 
 COMPONENT_INSTALL_ENV+= GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
 
+COMPONENT_TEST_ENV += CC=$(CC)
+COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
+COMPONENT_TEST_TRANSFORMS += \
+    '-ne "/^\# TOTAL:/p" ' \
+    '-ne "/^\# PASS:/p" ' \
+    '-ne "/^\# SKIP:/p" ' \
+    '-ne "/^\# XFAIL:/p" ' \
+    '-ne "/^\# FAIL:/p" ' \
+    '-ne "/^\# XPASS:/p" ' \
+    '-ne "/^\# ERROR::/p" '
+
 # common targets
-build:		$(BUILD_32_and_64)
+build:      $(BUILD_32_and_64)
 
-install:	$(INSTALL_32_and_64)
+install:    $(INSTALL_32_and_64)
 
 # To prevent "libtool_install_magic: unbound variable"
 unexport SHELLOPTS
 
-# TEST_32 fails test-math
-test:		$(TEST_32_and_64)
+test:       $(TEST_32_and_64)
 
```

Comment:
Please mark the generated dependencies with a "# Auto-generated dependencies" comment like
gmake REQUIRED_PACKAGES automatically do. Hand-crafted dependencies should be separated (probably with its own comment).

Reply:
They were gmake REQUIRED_PACKAGES but with the 
"# Auto-generated dependencies" deleted.

Just add comment back?